app/assess/models/round.py

```diff
@@ -1,33 +1,45 @@
+import inspect
 from dataclasses import dataclass
-from datetime import datetime
 from typing import List
 
 from .application import Application
 
 
 @dataclass
 class Round:
-    title: str
-    identifier: str
+    id: str
+    assessment_criteria_weighting: list
+    assessment_deadline: str
+    deadline: str
     fund_id: str
-    opens: datetime
-    deadline: datetime
-    eligibility_criteria: dict
+    opens: str
+    title: str
+    short_name: str
     applications: List[Application] = None
 
+    @classmethod
+    def from_dict(cls, d: dict):
+        # Filter unknown fields from JSON dictionary
+        return cls(
+            **{
+                k: v
+                for k, v in d.items()
+                if k in inspect.signature(cls).parameters
+            }
+        )
+
     @staticmethod
     def from_json(data: dict):
-        eligibility_criteria = {}
-        if "eligibility_criteria" in data:
-            for key, value in data["eligibility_criteria"].items():
-                eligibility_criteria.update({key: value})
+        # eligibility_criteria = {}
```

Comment:
I've left it there for now as the whole data structure is still being debated and not sure if this is in there or not, so it will remind us to sort it out I think once the structure is known.

Reply:
fair enough 👍